graphql/util.go

```diff
@@ -574,13 +574,40 @@ func isTaskBlocked(ctx context.Context, at *restModel.APITask) (*bool, error) {
 	return &isBlocked, nil
 }
 
+func isExecutionTask(ctx context.Context, at *restModel.APITask) (*bool, error) {
+	t, err := task.FindOneId(*at.Id)
+	if err != nil {
+		return nil, ResourceNotFound.Send(ctx, err.Error())
+	}
+	if t == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("task %s not found", *at.Id))
+	}
+	displayTask, err := t.GetDisplayTask()
```

Comment:
Instead of doing this, you can call t.IsPartOfDisplay. With both of these changes I think you also won't need to have a separate function for this